BookPlayer/Library/ItemListViewController.swift

```diff
@@ -24,16 +24,37 @@ class ItemListViewController: UIViewController, ItemList, ItemListAlerts, ItemLi
 
     private var previousLeftButtons: [UIBarButtonItem]?
     lazy var selectButton: UIBarButtonItem = UIBarButtonItem(title: "Select All", style: .plain, target: self, action: #selector(selectButtonPressed))
+    lazy var searchController: UISearchController = {
+        let searchController = UISearchController()
+        searchController.obscuresBackgroundDuringPresentation = false
+        searchController.searchResultsUpdater = self
+        return searchController
+    }()
 
     var library: Library!
 
-    // TableView's datasource
     var items: [LibraryItem] {
         guard self.library != nil else {
             return []
         }
 
-        return self.library.items?.array as? [LibraryItem] ?? []
+        let items: [LibraryItem] = self.library.items?.array as? [LibraryItem] ?? []
+
+        if let searchText = self.searchController.searchBar.text,
```

Comment:
Similar thing as with the first recommendation in the `PlaylistViewController` implementation